project/set_data_lat_long.py

```diff
@@ -18,4 +18,5 @@
 
 # print(sources_df["desc"].head(25))
 # Create csv file from dataframe
-sources_df.to_csv("all_sources_geoed.csv")
\ No newline at end of file
+# sources_df.to_csv("all_sources_geoed.csv")
+print(sources_df.sort_values(by=sources_df["date"], axis=1, ascending=True))
```

Comment:
Are we sure we want the ascending=True argument (where I think head will be the earliest incidents) instead of ascending=False?

Reply:
Something I missed. Thanks for the catch! I'll go back and fix that.